packages/shared/web/src/email-capture-section/input-form.tsx

```diff
@@ -0,0 +1,126 @@
+import { styled, useTheme, Box } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import { ProgressIndicator, Button, ButtonText } from '@notional-finance/mui';
+import { colors } from '@notional-finance/styles';
+import { CheckmarkRoundIcon } from '@notional-finance/icons';
+
+export const InputForm = ({ submitState, onInputChange, onSubmit, email }) => {
+  const theme = useTheme();
+
+  return (
+    <Box>
+      {submitState === 'pending' && (
+        <ProgressIndicator type="circular" size={32} />
+      )}
+      {submitState === 'success' && (
```

Comment:
Does this need to be capitalized?

Reply:
It doesn't have to be. I thought it worked to have it match the "YOUR EMAIL" LabelText that you see before you submit

Reply:
Lmk if you want me to change it

Reply:
All good, as long as it works